library/src/main/java/com/bumptech/glide/request/transition/DrawableCrossFadeTransition.java

```diff
@@ -57,4 +57,9 @@ public boolean transition(Drawable current, ViewAdapter adapter) {
     adapter.setDrawable(transitionDrawable);
     return true;
   }
+
+  @Override
+  public void cancel() {
+    // TODO
```

Comment:
https://developer.android.com/reference/android/graphics/drawable/TransitionDrawable.html#resetTransition()

Then swap the positions of the Drawables? I think?

Reply:
Well I don't know what would be expected as a cancel don't really know on cancel what image should be shown. But if you say I should show the last I'll do that :)